projects/core/internal/primitive-year-month-pagination/primitive-year-month-pagination.template.html

```diff
@@ -11,6 +11,7 @@
     </ng-container>
     <ng-template #button>
         <button
+            id="year-btn"
```

Comment:
Duplicate IDs are not valid HTML. I also would like to avoid adding attributes just for tests as it increases the bundle size.